clang/include/clang/CConv/ConstraintVariables.h

```diff
@@ -101,9 +101,12 @@ class ConstraintVariable {
   virtual bool hasWild(EnvironmentMap &E) = 0;
   virtual bool hasArr(EnvironmentMap &E) = 0;
   virtual bool hasNtArr(EnvironmentMap &E) = 0;
+
+  // Force use of equality constraints in function calls for this CV
   virtual void equateArgumentConstraints(ProgramInfo &I) = 0;
-  virtual void brainTransplant(ConstraintVariable *) = 0;
 
+  // Update this CV with information from duplicate declaration CVs
+  virtual void brainTransplant(ConstraintVariable *) = 0;
   virtual void mergeDeclaration(ConstraintVariable *) = 0;
```

Comment:
The changes here are formatting changes that I didn't stage properly; sorry for the noise!